ddtrace/contrib/langchain/patch.py

```diff
@@ -567,6 +567,7 @@ def traced_embedding(langchain, pin, func, instance, args, kwargs):
     """
     try:
         input_texts = get_argument_value(args, kwargs, 0, "texts")
+        chunk_size = get_argument_value(args, kwargs, 1, "chunk_Size")
```

Comment:
Just curious, which langchain embedding models is `chunk_size` used as an argument for? I see that it is present for `OpenAIEmbeddings.embed_documents()` but if it is not present for all embedding models signatures I would be careful adding this.

Reply:
I believe it's only the openAIembeddings but i thought that was the only type we supported anyways 🤔